src/ui/media-devices.js

```diff
@@ -78,17 +75,17 @@ function MediaDevices(props) {
         />
       )}
 
-      {supportsUserCapture() && (
+      {isUserCaptureSupported() && (
         <MediaDevice
           onClick={requestUserMedia}
-          title={t('share-webcam')}
+          title={onMobileDevice() ? t('share-camera') : t('share-webcam')}
```

Comment:
I do not think that this is necessary. Always use one of the strings.

Reply:
Lars mentioned that too. I still think "webcam" is slightly more clear on desktop, but I changed it now.